README.md

```diff
@@ -24,12 +24,15 @@ Notes:
 2. Since 0.6.0, HTTP becomes the default protocol.
 3. Since 0.7.0, gRPC is deprecated and not recommended, it may be removed in the future.
 4. Since 0.8.0, gRPC is removed.
+5. Since 0.8.0, the project groupId is rename from `com.github.housepower` to `com.clickhouse.spark`, and
+   the class `xenon.clickhouse.ClickHouseCatalog` is renamed to `com.clickhouse.spark.ClickHouseCatalog`
```

Comment:
it's important for users who want to upgrade, should also be mentioned in the release notes